bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractComponentImpl.java

```diff
@@ -35,7 +35,6 @@
 import com.adobe.cq.wcm.core.components.models.Component;
 import com.adobe.cq.wcm.core.components.models.datalayer.ComponentData;
 import com.adobe.cq.wcm.core.components.models.datalayer.builder.DataLayerBuilder;
-import com.adobe.cq.wcm.core.components.util.ComponentUtils;
```

Comment:
We have a `ComponentUtils` in this folder, and therefore any reference to the wcm one needs to be explicitly mentioned by path IMO.